tests/xtro-sharpie/tvOS-UserNotifications.ignore

```diff
@@ -0,0 +1,7 @@
+## Deprecated
+!missing-selector! UNMutableNotificationContent::setSummaryArgument: not bound
+!missing-selector! UNMutableNotificationContent::setSummaryArgumentCount: not bound
+!missing-selector! UNMutableNotificationContent::summaryArgument not bound
+!missing-selector! UNMutableNotificationContent::summaryArgumentCount not bound
+!missing-selector! UNNotificationContent::summaryArgument not bound
+!missing-selector! UNNotificationContent::summaryArgumentCount not bound
```

Comment:
why were the deprecation attributes were only added for iOS ?

Reply:
I only added the deprecation attribute to iOS because TV and Watch was not supported in these selectors. I thought this would be better than introducing them and deprecating them in this update. What do you think?

Reply:
make sense :)
In such cases you can add a comment like `# both added and deprecated in Xcode 13 beta 1`
It helps reviews and can be useful in the future (where the wiki data might not be available anymore)

Reply:
Makes sense! I'll update it :)